File: excelToMongoProcess/FileToMongo/FileToMongo/Vocabulary.cs
using System;
using System.Collections.Generic;
using System.Linq;
using System.Text;
using System.Threading.Tasks;

namespace FileToMongo
{
    public class Vocabulary
    {
        public string Name { get; set; }
        public string Synonyms { get; set; }
        public string Translate { get; set; }
        public string Examples { get; set; }
        public string Meanings { get; set; }
        public string Type { get; set; }
    }
}

File: backend/models/vocabulary.js
'use strict'

var mongoose = require('mongoose');
var schema = mongoose.Schema;

var VocabularySchema = schema({
    name: String,
    synonyms: String,
    translate: String,
    examples: String,
    meanings: String,
    type: String

});

module.exports = mongoose.model('vocabularies', VocabularySchema);
